docs/general/application.rst

```diff
@@ -159,7 +159,7 @@ Array, with each element containing the following properties; read-only:
 ``displayName``  String representing the localized display name of the
                  effect as seen in the Effect menu.
 ``category``     String representing the localized category label as seen
-                 in the Effect menu. This can be "" for synthetic effects
```

Comment:
This isn't a typo, it's an empty string.

Reply:
Sorry about that. I'll revert that part.